.github/workflows/update-sdk.yml

```diff
@@ -0,0 +1,21 @@
+name: update-dotnet-sdk
+
+on:
+
+  # Scheduled trigger to check for .NET SDK updates at 12 UTC every Monday
+  schedule:
+    - cron:  '00 12 * * MON'
+
+  # Manual trigger to update the .NET SDK on-demand.
+  workflow_dispatch:
+
+jobs:
+  update-dotnet-sdk:
+    name: Update .NET SDK
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: martincostello/update-dotnet-sdk@v3.1.0
```

Comment:
- uses: martincostello/update-dotnet-sdk@8765302da88e1a78c845125626f3726190908593 # v3.1.0

Commit is from https://github.com/martincostello/update-dotnet-sdk/commit/8765302da88e1a78c845125626f3726190908593